netty/src/test/java/io/grpc/netty/NettyHandlerTestBase.java

```diff
@@ -203,6 +203,10 @@ public int compareTo(Delayed o) {
     }
   }
 
+  protected final WriteQueue writeQueue() {
```

Comment:
This method should be final because `initChannel()` is final, also `enqueue()` is final.

Reply:
This method is final...